_tour/pattern-matching.md

```diff
@@ -143,7 +142,33 @@ def findPlaceToSit(piece: Furniture): String = piece match {
 ```
 This is useful for pattern matching because we don't need a "catch all" case.
 
+## Summary of pattern matching syntax
+
+```tut
+case class Point(x: Int, y: Int)
+
+val value: Any = "test"
+
+val res: String = value match {
+  case "a" => "exact match"
+  case v @ "b" => "exact match: " + v
```

Comment:
The way I look at the tour is as a showroom, not really an exhaustive reference. I think pattern binders have not been introduced earlier (and I don’t think they should) so I don’t think we should have them in a summary. Also, I would rather put a link to a more comprehensive documentation (that we don’t have, I believe…) than putting a summary like this one here.

But maybe others have a different opinion?

Reply:
I understand your point of view.